main.js

```diff
@@ -219,6 +220,53 @@ app.get("/auth/google/callback",
     }
 );
 
+//sends text to dialogflow
+app.post("/sendToDialogflow", function(req, res){
+	//create variables
+	const projectId = 'helloworld-b026e';
+	const sessionId = '123';
+	const query = req.body["content"];
+	//const query = req.body["content"];
```

Comment:
I think you can get rid of this commented out code, since it is un-commented above

Reply:
good catch....yeah it was there from when i was testing different values for the query variable. i'll update that and push it to this PR.